CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.10.2)
```

Comment:
CMake uses 2-space indents.

Reply:
Not sure what this is about? Are you saying the indents in the file should be 2 spaces note 4?

Reply:
No sure this is terribly important but including it in the editorconfig just to be thorough.